.vsts-pipelines/builds/ci-public.yml

```diff
@@ -20,23 +20,94 @@ phases:
       displayName: Use Node 8.x
       inputs:
         versionSpec: 8.x
+    - script: npm install -g selenium-standalone
```

Comment:
It looks like this is the same content as ci-internal.yml. Can you move this shared content to a template file in this repo and import it?